### PR TITLE
I added a guide for the windows installation

### PR DESCRIPTION
**Pull request by TheFibonacciEffect**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,27 @@ These are the final system requirements: `Cairo`, `Latex`, `ffmpeg` and `sox`.
 
 ### Windows Users
 
+!!!to my knowledge manim only works with python 3.7 (yet)!!!
```
Comment (TheFibonacciEffect) [inline README.md:67]: is this correct or is my knowledge outdated?

Comment (CodeForeverAndEver) [reply README.md:67]: I believe it is possible to do with 3.8 but not super easily as you will have  to `pip install` a bunch of different packages.

Comment (TheFibonacciEffect) [reply README.md:67]: Oh, okay, so well just write that it is only for 3.7 or something and the guide is not guranteed to work in later versions

Comment (CodeForeverAndEver) [reply README.md:67]: 👍 Right, and then later, it can be updated. Your tutorial looks accurate as well.

**Comment by TheFibonacciEffect** [comment]
Could you merge it then?

**Comment by TheFibonacciEffect** [comment]
You will need to approve it, so it is merged

**Comment by CodeForeverAndEver** [comment]
@TheFibonacciEffect I tried, maybe it didn't go through?

**Pull request merged**